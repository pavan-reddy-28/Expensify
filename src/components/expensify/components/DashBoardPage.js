import React from 'react'
import ExpenseList from './ExpenseList'

import ExpenseFilterInput from './ExpenseListFilters'
function DashBoardPage() {
    return (<>
        <ExpenseFilterInput/>
       <ExpenseList/>
       </>
    )
}

export default DashBoardPage
