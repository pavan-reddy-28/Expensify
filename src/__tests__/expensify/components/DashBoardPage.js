import React from 'react'
import {shallow} from 'enzyme'
import DashBoardPage from '../../../components/expensify/components/DashBoardPage'

test('should render dashboard page', () => {
    const wrapper = shallow(<DashBoardPage/>)
    expect(wrapper).toMatchSnapshot();
})
