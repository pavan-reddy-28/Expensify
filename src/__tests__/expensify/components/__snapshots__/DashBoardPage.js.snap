// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard page 1`] = `
<Fragment>
  <Connect(ExpenseFilterInput) />
  <Connect(ExpenseList) />
</Fragment>
`;
